### DISCUSSION TITLE
Syncing a single directory in production

### DISCUSSION THREAD

**Discussion by jeritiana** [abraunegg/onedrive#1859] [Q&A] [answered]
Hi there,

I would like to sync only a single directory, in a permanent way.
I.e. skip all dirs on the OneDrive remote except one which has a local copy and is continuously synced.

The things I thought of:

* I saw there is a `--synchronize --single-directory` [option](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-directory-sync) but if I understand correctly, this is intended for a one-time manually run job, while what I would want is something with `--monitor`
* Or maybe there is an _except_ syntax in the [`skip_dir`](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#skip_dir) config? i.e. skip all other dirs except the matching one

What would be the best way to achieve this?

Thanks all

**Comment by abraunegg** [comment, marked as answer]
@jeritiana 
> What would be the best way to achieve this?

You can use `--single-directory` with `--monitor` when running manually, and you can also modify any systemd startup script to include `--single-directory` - thus giving you your desired outcome.

> Or maybe there is an except syntax in the [skip_dir](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#skip_dir) config? i.e. skip all other dirs except the matching one

skip_dir is for skipping directories. To do what you are wanting in this fashion you want to use sync_list where you can specify what to include and exclude from syncing.

**Reply by jeritiana** [reply]
Great, thank you very much @abraunegg!
